.github/ISSUE_TEMPLATE/content_issue_report.yml

```
@@ -0,0 +1,31 @@
+name: Content Issue Report
+description: Report a content issue on the website.
+labels: [bug]
```

```suggestion
labels: [content issue]
```
I've added a new label for this.